src/deploy/functions/backend.ts

```diff
@@ -204,6 +204,8 @@ export interface ServiceConfiguration {
 
 export type FunctionsPlatform = "gcfv1" | "gcfv2";
 
+export type SecurityLevel = "SECURE_ALWAYS" | "SECURE_OPTIONAL";
```

Comment:
nit* you might want to use type definition define din cloudfunctions.ts instead.